emis-client/pom.xml

```diff
@@ -12,6 +12,9 @@
   <artifactId>emis-client</artifactId>
   <properties>
     <github.repository-name>lighthouse-enterprise-military-information-service</github.repository-name>
+    <wiremock.version>2.27.2</wiremock.version>
+    <jacoco.coverage>0.10</jacoco.coverage>
+    <maven-surefire-plugin.version>3.0.0-M7</maven-surefire-plugin.version>
```

Comment:
Why specify this version? it should pick the matched version from the parent hierarchy's plugin management.

Reply:
yeah, if I go and use the version defined in the parent, it's 2.22.2, which normally works just fine for the other clients. For some reason it doesn't run the unit tests here. I don't know what the root error is... specifying the version here like this is basically a bandaid for the fact that I can't figure out why 2.22.2 won't run the tests.

Reply:
It's because of some dependency shenanigans. You will want to get rid of [these two](https://github.com/department-of-veterans-affairs/lighthouse-enterprise-military-information-service/pull/17/files#diff-194cea87aeaad16e8d3cd412967a56326ddf27943460ce7c14b6ebdd017d2aa5R29-R38) and add in
```
 <dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-starter-test</artifactId>
      <scope>test</scope>
    </dependency>
```
And it looks like that is working for me.

Reply:
works for me too, thanks. I wouldn't have thought of doing that.